filebeat/docs/getting-started.asciidoc

```diff
@@ -99,9 +99,32 @@ To see the full list of variables for a module, see the documentation under
 
 include::{libbeat-dir}/shared/config-check.asciidoc[]
 
+[float]
+[[setup-application-logs]]
+=== Step 4: Ingest application logs
+
+This section is about best practices for ingesting logs from your own applications,
+as well as for off-the-shelf applications for which we don't have suitable built-in modules.
+
+Although you can ingest raw plain-text logs, we recommend you structure
```

Comment:
I would just state what we want users to do without saying "we recommend." I'm assuming that ingesting raw plain-tex logs is a corner case, not a common use case.